docs/developer-docs/multi-chain/ethereum/evm-rpc/overview.mdx

```diff
@@ -0,0 +1,83 @@
+---
+keywords: [advanced, ethereum, evm rpc, evm, tutorial]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Overview
+
+<MarkdownChipRow labels={["Advanced", "Ethereum", "Tutorial"]} />
+
+**EVM RPC** is an ICP canister smart contract for communicating with [Ethereum](https://ethereum.org/en/) and other [EVM blockchains](https://chainlist.org/?testnets=true) using an on-chain API. Communication with EVM-compatible chains includes obtaining and sending information such as:
+
+- The logs of a specific block or transaction.
+
+- Information about a block.
+
+- Details about a transaction.
+
+- Historical data regarding the network's gas fee.
+
+- Transactions for a specific address.
+
+- Submitting a signed transaction.
+
+The source code for this project is available on GitHub ([internet-computer-protocol/evm-rpc-canister](https://github.com/internet-computer-protocol/evm-rpc-canister) ⭐️) under the Apache 2.0 license.
+
+## Goal
+
+The goal of the EVM RPC canister is to act as a gateway between canisters on ICP and EVM-compatible networks through RPC services such as Alchemy, Ankr, or Cloudflare. The EVM RPC canister provides the necessary facilities to send requests to the Ethereum JSON-RPC API and receive responses that are immediately useful in the context of a canister.
+
+This canister is available for all dapp developers to use and adopts a decentralized governance model through the NNS.
```

Comment:
i'm not sure i understand this sentence 🤔

Reply:
revised this sentence